### ISSUE TITLE
Suggestion: Header for thermo.out file

### ISSUE THREAD

**Issue by erhart1** [brucefan1983/GPUMD#473]
It would be useful to include a header in the `thermo.out` file that specifies the meaning of the columns.
If these lines are prepended by a widely used comment symbols (e.g., `#`), the user experience should be (hopefully) unaffected.
At the same time it would help users (and parsers) to read the content of the file.
This would in particular improve readability/interpretability if the number of columns/data changes, e.g., when switching from an orthorhombic to a triclinic cell.
Finally, this would also be useful if one were to decides to add more information to this file in the future.

**Comment by hityingph** [comment]
Yes, I agree with. Acutally, I also suggested this before, however, Zheyong @brucefan1983 prefered the pure arrays output without strings (header).

**Comment by brucefan1983** [comment]
Perhaps we can do this for all the output files in GPUMD-v4.0, for which we will introduce breaking changes?

**Comment by brucefan1983** [comment]
finished in PR #975
